src/install-pnpm/run.ts

```diff
@@ -1,3 +1,4 @@
+import * as core from '@actions/core'
```

Comment:
So we are going to bundle all of `core`? This would increase the size of `dist/index.js`. I think it is better to `import { addPath, exportVariable }` separately.